CODE_OF_CONDUCT.md

```diff
@@ -2,24 +2,26 @@
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion,  sexual identity and orientation, or other factors.
```

Comment:
This is pretty vague. Who decides what those other factors are?

Reply:
Conduct@ like with everything else; it seems counter-intuitive to have a limited list.

I have no issue reverting this.

Reply:
Agreed. We do not wish to be vague on this.